### DISCUSSION TITLE
Why isn't TransformersRecognizers part of lib?

### DISCUSSION THREAD

**Discussion by danielsaaf** [microsoft/presidio#1055] [Q&A]
Hi!

In the docs i found a really useful class (TransformersRecognizer) https://github.com/microsoft/presidio/blob/aeaff636a6922757f169e297c0dd0333dc722343/docs/samples/python/transformers_recognizer/transformer_recognizer.py
Which allows me to setup at transformer-recognizer based on a huggingface-model according to the documentation  under /docs/samples/python 

My question is, why isn't this implemented in the library to enable users to import that directly? 
Are there any recommendation on how to handle it? Would it be possible to import it from the docs-source?

**Comment by omri374** [comment]
Hi @danielsaaf,
While the TransformsRecognizer is not officially part of the list, there's an ability to introduce transformer models using the TransformerNlpEngine. More info can be found here: https://microsoft.github.io/presidio/analyzer/nlp_engines/transformers/
We leave the decision on whether to use the `TransformersNlpEngine` or the `TransformersRecognizer` to the user as there are pros and cons for each approach.
